src/modules/AieraChat/components/Messages/MessageFactory/index.tsx

```diff
@@ -7,17 +7,21 @@ import { MessagePrompt } from './MessagePrompt';
 import { MessageResponse } from './MessageResponse';
 import { SourcesResponse } from './SourcesResponse';
 
+/* eslint-disable react/prop-types */
```

Comment:
it seems to have an issue with the VirtuosoMessageListProps (even though nothing changed around that during this PR), so disabling this check